tools/quantification/.shed.yml

```diff
@@ -0,0 +1,6 @@
+owner: perssond
+name:  quantification
```

Comment:
@alliecreason I recommend to give better names for the TS repo and as tool IDs. It's a very genric name and admjs, as well as users interact with the ID.